### PR TITLE
Small textual changes to clarify role of Foundation For Public Code

### PR DESCRIPTION
**Pull request by felixfaassen**
Small textual changes to clarify the role of the Foundation For Public Code as a facilitator rather than a development partner.

@bvhme could you review the proposed changes?

### CODE REVIEW

**Comment by jaccobrouwer** [comment]
I noticed that some of sentences are not correct. The suggested changes are in essence fine, but the overall text needs some adjustment as well


Jacco Brouwer

Programma manager/PO Signalen


Vereniging Nederlandse gemeenten


M +31 6 53 621 906

https://www.linkedin.com/in/jaccobrouwernl/

Skype/jaccobrouwer1

www.VNG.nl<http://www.VNG.nl>


________________________________
Van: Felix Faassen <notifications@github.com>
Verzonden: woensdag 14 oktober 2020 15:45
Aan: Signalen/signalen.org <signalen.org@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Onderwerp: [Signalen/signalen.org] Small textual changes to clarify role of Foundation For Public Code (#42)


Small textual changes to clarify the role of the Foundation For Public Code as a facilitator rather than a development partner.

@bvhme<https://github.com/bvhme> could you review the proposed changes?

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/Signalen/signalen.org/pull/42

Commit Summary

  *   Small textual changes to clarify role of Foundation For Public Code

File Changes

  *   M src/pages/common-ground.md<https://github.com/Signalen/signalen.org/pull/42/files#diff-ab126b54fd44bd166cbe3ae6d53a35596121e376ea7439ac3c81fba4fe917d23> (2)
  *   M src/pages/over-signalen.md<https://github.com/Signalen/signalen.org/pull/42/files#diff-0de0d42c8452e476141e3d2c42cf9d35db4ea5b1d87eb870a0f5b5aaab97add7> (3)
  *   M src/pages/roadmap.js<https://github.com/Signalen/signalen.org/pull/42/files#diff-686033ea94e1480400dae4db81bd9cf11732e7f7e0c2d4c711222f1d322dc76e> (2)

Patch Links:

  *   https://github.com/Signalen/signalen.org/pull/42.patch
  *   https://github.com/Signalen/signalen.org/pull/42.diff

File: src/pages/over-signalen.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ template: "default"
 
 ## Samenwerken aan Signalen
```
Comment (bartjkdp) [inline src/pages/over-signalen.md:6]: The page /over-signalen is not available from the website. I think we should remove this file from the repository.

File: src/pages/roadmap.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ const RoadmapPage = () => (
           <Timeline.Content>
             <Span fontSize="0.9rem">2 maart 2020</Span>
             <Heading as="h3" fontSize="1.5rem">Kopgroep Signalen</Heading>
-            <p>‘s-Hertogenbosch, Amsterdam, Almere, VNG Realisatie en de Foundation for Public Code vormen samen de Kopgroep met als doel om Signalen bruikbaar te maken voor andere gemeenten.</p>
+            <p>‘s-Hertogenbosch, Amsterdam, Almere, VNG Realisatie vormen samen de Kopgroep gefaciliteerd door de Foundation for Public Code met als doel om Signalen bruikbaar te maken voor andere gemeenten.</p>
```
Comment (jaccobrouwer) [inline src/pages/roadmap.js:59]:
```suggestion
            <p>‘s-Hertogenbosch, Amsterdam, Almere, VNG Realisatie vormen samen de Kopgroep, gefaciliteerd door de Foundation for Public Code met als doel om Signalen bruikbaar te maken voor andere gemeenten.</p>
```

File: src/pages/common-ground.md [hunk 3]
```diff
@@ -21,6 +21,6 @@ De code van Signalen is openbaar en kun je niet zozeer spreken over eigenaarscha
 
 ### Samen werken aan Signalen
 
-In samenwerking met VNG Realisatie, gemeente Amsterdam, ’s-Hertogenbosch, Almere en The Foundation for Public Code is Signalen toegankelijk gemaakt voor iedere gemeente in Nederland.
+In samenwerking met VNG Realisatie, gemeente Amsterdam, ’s-Hertogenbosch, Almere en gefaciliteerd door de Foundation for Public Code is Signalen toegankelijk gemaakt voor iedere gemeente in Nederland.
```
Comment (jaccobrouwer) [inline src/pages/common-ground.md:24]:
```suggestion
In samenwerking met VNG Realisatie, gemeente Amsterdam, ’s-Hertogenbosch, Almere en gefaciliteerd door de Foundation for Public Code, is Signalen toegankelijk gemaakt voor iedere gemeente in Nederland.
```

File: src/pages/over-signalen.md (hunk 1 above)
Comment (felixfaassen) [reply src/pages/over-signalen.md:6]: i'll leave the file in this PR we can delete it seperately

**Comment by felixfaassen** [comment]
@jaccobrouwer can you approve this PR?

**Pull request merged**